changelog.txt

```diff
@@ -29,6 +29,7 @@ can then iterate over a channel group, which yields all channels for a given gro
 loaded again when reading in the Detector from a nur file
 - Added LOFAR coordinates to Detector site coordinates
 - Implemented mattak dataset iterator in readRNOGDataMattak.run()
+- Fixed bug in phased-array noise generation script
```

Comment:
can you specify what exactly was wrong? Something like: "Previous version always generated triggered noise that triggered in half of the beams"